cfWorkflow/cfWorkflow.go

```diff
@@ -81,8 +81,16 @@ func (c *cfWorkflow) Setup(ccg cfCmdGenerator.CfCmdGenerator) []cmdStartWaiter.C
 		ccg.Api(c.cf.API),
 		ccg.Auth(c.cf.AdminUser, c.cf.AdminPassword),
 		ccg.CreateOrg(c.org),
-		ccg.CreateSpace(c.org, c.space),
 	}
+
+	if c.cf.IsolationSegment != "" {
+		ret = append(ret,
+			ccg.EnableOrgIsolation(c.org, c.cf.IsolationSegment),
+			ccg.SetOrgDefaultIsolationSegment(c.org, c.cf.IsolationSegment),
+		)
+	}
+	ret = append(ret, ccg.CreateSpace(c.org, c.space))
```

Comment:
Does the space need to be created after the org has been designated for an isolation segment? Like, if the flow was create org -> create space -> designate org for an isolation segment, would the space still be designated for an isolation segment? Just curious.